src/david/UrlCapture.py

```diff
@@ -0,0 +1,15 @@
+def urlCapture(text):
+    urls = []
+    n = len(text)
+    indexstart = 0
+    while n >= indexstart:
+        indexstart = text.find('http://', indexstart)
+        if indexstart < 0:
+            break
+        iindexend = text.find(' ', indexstart)
+        urls.append(text[indexstart:iindexend])
+        indexstart = iindexend
+    print (" ".join(str(x) for x in urls))
```

Comment:
if I remember well the exercise require to return the urls found, not only print

please add a return